docs/tutorials/mongoid-documents.txt

```diff
@@ -176,6 +177,38 @@ Types that are not supported as dynamic attributes since they cannot be cast are
 - ``DateTime``
 - ``Range``
 
+The StringifiedSymbol Type
+--------------------------
+
+When using the built-in Symbol field type, Mongoid defaults to serializing the field into the BSON::Symbol
+type. For more information on the BSON::Symbol type, see
+`here <https://docs.mongodb.com/mongoid/master/tutorials/mongoid-documents/#bson-symbol-type>`_.
+BSON::Symbol is deprecated, so the StringifiedSymbol type allows users to write code
+that makes use of the built-in Symbol type while ensuring interoperability with other drivers.
+The StringifiedSymbol type stores all data on the database as a String, while returning values to the
+user as a Symbol.
+
+An example usage is shown below:
+
+.. code-block:: ruby
+
+  class Post
+    include Mongoid::Document
+
+    field :status, type: StringifiedSymbol
+  end
+
+  post = Post.new(status: "hello")
```

Comment:
In this example, could you pass "hello" as a symbol, since that's probably how the user would want to use it?

Reply:
I agree that the first example would make more sense with `:hello`, but this example could be copied/moved under the next paragraph that talks about values being converted to symbols when returned to application.